File: C_Sharp_code/Example/WindowsFormsApp/VB_demo07/Form2.vb
Public Class Form2
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        If textBox1.Text > 100 Or textBox1.Text < 0 Then
            MessageBox.Show("Score Error")
        Else
            MessageBox.Show("Score:" + textBox1.Text)
        End If
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo07/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo07
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int s = Convert.ToInt32(textBox1.Text);
            if (s >100 || s<0)
            {
                MessageBox.Show("Score Error");
            }else
                MessageBox.Show("Score:"+s.ToString());
        }
    }
}
